dev-client/src/theme.ts

```diff
@@ -408,6 +413,12 @@ export const theme = extendTheme({
           fontWeight: 500,
           lineHeight: '32px',
         },
+        h6Italic: {
+          fontSize: '20px',
+          fontWeight: 500,
+          lineHeight: '32px',
+          fontStyle: 'italic',
+        },
```

Comment:
suggestion: since `h6Italic` is identical to `h6` other than `fontStyle: 'italic'`, do you think this might be an appropriate place to pass this style directly to the heading rather than having a separate variant for it? (otherwise would we have separate variants of `h6Bold` `h6BoldItalic` `h5Italic` etc etc?)

Reply:
Strong agree with @shrouxm's suggestion.